src/schema/digital-service/credential-digital-service-execution-result.ttl

```diff
@@ -0,0 +1,75 @@
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <https://ontology.okp4.space/schema/credential/digital-service/execution-proof/> .
+@prefix schema: <http://schema.org/> .
+
+:DigitalServiceExecutionResult a rdfs:Class ;
+  rdfs:label "Digital Service Execution Result"@en ;
+  rdfs:comment """
+  Captures the details of the result of the execution of a digital service.
+  """@en .
+
+:DigitalServiceExecutionResultCredential a rdfs:Class ;
+  rdfs:label "Digital Service Execution Result Credential"@en ;
+  rdfs:comment """
+  A credential that provides proof of the execution of a digital service for a considered Service Execution Order. It includes details of the service that was executed, the time and date of execution, the parameters used, and any outcomes or results relevant to the execution.
+  """@en .
+
+:hasExecutionResult a rdf:Property ;
+  rdfs:label "has execution result"@en ;
+  rdfs:comment """
+  The execution result of a digital service execution order.
+  """@en ;
+  schema:domainIncludes :DigitalServiceExecutionResultCredential ;
+  schema:rangeIncludes :DigitalServiceExecutionResult .
+
+:hasExecutionStatus a rdf:Property ;
+  rdfs:label "has execution status"@en ;
+  rdfs:comment """
+  Indicates the status of the service execution.
+  """@en ;
+  schema:domainIncludes :DigitalServiceExecutionResultCredential ;
+  schema:rangeIncludes <https://ontology.okp4.space/thesaurus/digital-service-execution-status> .
+
+:hasExecutionTime a rdf:Property ;
+  rdfs:label "has execution time"@en ;
+  rdfs:comment """
+  The time and date when the Digital Service was executed.
+  """@en ;
+  schema:domainIncludes :DigitalServiceExecutionResult ;
+  schema:rangeIncludes schema:DateTime .
```

Comment:
For :DigitalServiceExecutionOrder you put both Date and DateTime. Should we have both here too ?

Reply:
That's a good question. Let's say that in the first case (`:DigitalServiceExecutionOrder`) I've left the two type of Dates because the precision isn't very important in fact, it's purely informative (the real contractual date is that of the submission in the protocol - the date of the transaction). For the execution result, on the other hand, the contractual date is the date in the VC, and it needs to be precise, hence `DateTime`. But it's quite subtle and we could align the 2 with DateTime only, I think.

Reply:
Fair enough thanks

Reply:
Okay. So I propose passing the PR as it is. We'll do the adjustment changes in another (there will certainly be others).

Reply:
Sounds good 👍🏻